### PR TITLE
Fix/login page changes

### PR DESCRIPTION
**Pull request by aayushah711**
- [x] Innovation Society Logo
- [x] CARE Logo. Push Coronasafe to the bottom of the page with the brief
- [x] Login page title should be 'MahaKavach'
- [x] The 'Central Resource Management Platform' should be a subtitle

### CODE REVIEW

**Comment by bodhish** [comment]
@aayushi-shah-sharechat wrong repo 😅

**Comment by bodhish** [comment]
@aayushah711 wrong repo 😅

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "quoteProps": "as-needed",
+  "requirePragma": false,
+  "semi": true,
+  "singleQuote": false,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "useTabs": false,
+  "vueIndentScriptAndStyle": false,
+  "filepath": "/Users/aayushishah/my-files2/care_fe/src/Components/Auth/Login.tsx"
+}
```
Comment (smartclash) [inline .prettierrc.json:20]: Don't add your prettier config. Do delete this.

File: src/Components/Shifting/BoardView.tsx [hunk 2]
```diff
@@ -54,61 +65,185 @@ export default function BoardView() {
...
           <button
-            className={"flex leading-none border-2 border-gray-200 rounded-full items-center transition-colors duration-300 ease-in focus:outline-none hover:text-blue-400 focus:text-blue-400 rounded-r-full px-4 py-2"
-              + (boardFilter === ACTIVE ? " bg-white text-gray-800" : " bg-gray-200 text-sm text-gray-500")}
-            onClick={_ => setBoardFilter(ACTIVE)}
+            className={
+              "flex leading-none border-2 border-gray-200 rounded-full items-center transition-colors duration-300 ease-in focus:outline-none hover:text-blue-400 focus:text-blue-400 rounded-r-full px-4 py-2" +
+              (boardFilter === ACTIVE
+                ? " bg-white text-gray-800"
+                : " bg-gray-200 text-sm text-gray-500")
+            }
```
Comment (smartclash) [inline src/Components/Shifting/BoardView.tsx:105]: Can we use `classnames` package instead?

File: src/Components/Shifting/BoardView.tsx [hunk 3]
```diff
@@ -54,61 +65,185 @@ export default function BoardView() {
...
           <button
-            className={"flex leading-none border-2 border-gray-200 rounded-full items-center transition-colors duration-300 ease-in focus:outline-none hover:text-blue-400 focus:text-blue-400 rounded-r-full px-4 py-2"
-              + (boardFilter === COMPLETED ? " bg-white text-gray-800" : " bg-gray-200 text-sm text-gray-500")}
-            onClick={_ => setBoardFilter(COMPLETED)}>
+            className={
+              "flex leading-none border-2 border-gray-200 rounded-full items-center transition-colors duration-300 ease-in focus:outline-none hover:text-blue-400 focus:text-blue-400 rounded-r-full px-4 py-2" +
+              (boardFilter === COMPLETED
+                ? " bg-white text-gray-800"
+                : " bg-gray-200 text-sm text-gray-500")
+            }
```
Comment (smartclash) [inline src/Components/Shifting/BoardView.tsx:116]: Can we use `classnames` package instead?

**Review by smartclash — CHANGES_REQUESTED** [review]

**Comment by aparnacoronasafe** [comment]
@bodhish What do we do about this? wrong repo. please close or deal appropriately

**Pull request closed without merging**